atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CellLoader.java

```diff
@@ -92,10 +93,13 @@ void loadWithTs(
             boolean loadAllTs,
             CassandraKeyValueServices.ThreadSafeResultVisitor visitor,
             ConsistencyLevel consistency) {
-        Map<InetSocketAddress, List<Cell>> hostsAndCells =
-                HostPartitioner.partitionByHost(clientPool, cells, Cell::getRowName);
+        Map<InetSocketAddress, List<Cell>> hostsAndCells;
+        try(CloseableTracer tracer = CloseableTracer.startSpan("partitionByHost")) {
```

Comment:
nit: checkstyle will probably want a space after 

Also we should probably include a bit of metadata (the table ref and cells.size(), maybe)?